CDS/scripts/atom_image.pl

```diff
@@ -30,12 +30,21 @@ sub is_imageurl_valid {
 print "INFO: Attempting to get image URL from CAPI\n";
 
 my $ua = LWP::UserAgent->new;
-my $req = $ua->request(GET 'https://internal.content.guardianapis.com/atom/media/'.$store->substitute_string($ENV{'atom_id'}));
+my $response = $ua->request(GET 'https://internal.content.guardianapis.com/atom/media/'.$store->substitute_string($ENV{'atom_id'}));
 
-my $capi = decode_json($req->content);
+if ($response->code == 503) {
+	my $response = $ua->request(GET 'https://internal.content.guardianapis.com/atom/media/'.$store->substitute_string($ENV{'atom_id'}));
+}
+
+if ($response->code != 200) {
+	print "-ERROR: Error when attempting to access CAPI \n";
+	exit(1);
+}
+
+my $capi = decode_json($response->content);
 
 unless (is_imageurl_valid($capi->{'response'}->{'media'}->{'data'}->{'media'}->{'trailImage'}->{'master'}->{'file'})) {
-	print "-ERROR: Unable to get URL. \n";
+	print "-ERROR: Unable to get URL. Got ".$capi->{'response'}->{'media'}->{'data'}->{'media'}->{'trailImage'}->{'master'}->{'file'}.". Malformed string\n";
```

Comment:
to be honest I would put this print line in `is_imageurl_valid` because if the capi data format changes this means we need to change in 2 places or the logging breaks. i'd rather change in only one and be confident that the logging will always show me the actual string tested

Reply:
Done